src/administration/web/email.md

```diff
@@ -10,12 +10,17 @@ These SendGrid sub-accounts are capped at 12k emails per month.
 You can use `/usr/sbin/sendmail` on your application container to send emails
 with the assigned SendGrid sub-account.
 
-Alternatively, you may use your own SMTP server.
-In this case, use port 465 / 587 since port 25 is blocked for security reasons.
+We do not guarantee the deliverability and support white-labeling of emails.
```

Comment:
delivery of the emails :)

regarding white-labeling, read https://sendgrid.com/docs/Classroom/Deliver/Delivery_Introduction/all_you_need_to_know_about_whitelabeling.html or https://sendgrid.com/blog/email-whitelabeling-why-its-important-and-how-sendgrid-helps/